tests.js

```diff
@@ -291,7 +294,8 @@ window.Specs = {
 			"linear-gradient()": {
 				"links": {
 					"tr": "#linear-gradients",
-					"dev": "#linear-gradients"
+					"dev": "#linear-gradients",
+					"mdn": "linear-gradient"
```

Comment:
This (and all other gradient functions) looks like it can be generated. Your code can just remove the `()` if the feature is a function.